ecl/plugins/org.eclipse.rcptt.ecl.core/src/org/eclipse/rcptt/ecl/internal/core/ForeachService.java

```diff
@@ -55,4 +58,11 @@ public IStatus service(Command command, IProcess context)
 		}
 		return status;
 	}
+
+	private static Command wrapBody(Command body) {
```

Comment:
The same question. Should we add one more val to the 'foreach' command definition to keep position into it? In the same way as it is done with 'repeat' command.